airbyte-webapp/src/core/domain/connectorBuilder/ConnectorBuilderProjectsRequestService.ts

```diff
@@ -0,0 +1,8 @@
+import { listConnectorBuilderProjects } from "core/request/AirbyteClient";
+import { AirbyteRequestService } from "core/request/AirbyteRequestService";
+
+export class ConnectorBuilderProjectsRequestService extends AirbyteRequestService {
+  public list(workspaceId: string) {
```

Comment:
FYI @timroes - you mentioned you wanted to refactor how the api service works, we are currently introducing new functionality here